app/decorators/article_decorator.rb

```diff
@@ -0,0 +1,15 @@
+class ArticleDecorator < SimpleDelegator
+  def self.wrap(collection)
```

Comment:
What do you think if we create `BaseDecorator` and move this method into this class?